### ISSUE TITLE
Form elements should have a visible label

### ISSUE THREAD

**Issue by OlgaLarina** [surveyjs/survey-creator#6954]
https://surveyjs.io/survey-creator/examples/covid-19-screening-form-template/reactjs

Form elements should have a visible label
Element location: #sq_147i
Element source: ```<input id="sq_147i" class="spg-input spg-text" type="text" placeholder="" aria-required="false" aria-labelledby="sq_147_ariaTitle" aria-describedby="sq_147_ariaDescription" aria-invalid="false" data-rendered="r">```

https://surveyjs.io/survey-creator/examples/product-feedback-survey-template/reactjs
https://surveyjs.io/survey-creator/examples/page-level-editing/reactjs

Form elements should have a visible label
Ensure that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes
Element location: #sq_147i
Element source: ```<input id="sq_147i" class="spg-input spg-text" type="text" placeholder="Ex.: 6in" aria-required="false" aria-labelledby="sq_147_ariaTitle" aria-describedby="sq_147_ariaDescription" aria-invalid="false" data-rendered="r">```

https://surveyjs.io/survey-creator/examples/customize-property-editors/reactjs
https://surveyjs.io/survey-creator/examples/dynamically-modify-newly-added-questions/reactjs#
https://surveyjs.io/survey-creator/examples/create-custom-adorners/reactjs

Form elements should have a visible label
Ensure that every form element has a visible label and is not solely labeled using hidden labels, or the title or aria-describedby attributes

Element location: #sq_260i_0
Element source: ```<input type="text" autocomplete="off" id="sq_260i_0" class="sd-dropdown__filter-string-input spg-dropdown__filter-string-input" role="combobox" aria-required="false" aria-invalid="false" aria-expanded="false" aria-labelledby="sq_260_ariaTitle" aria-describedby="sq_260_ariaDescription" aria-controls="sq_260i_list" placeholder="Select..." readonly="" inputmode="text">```

Element location: #sq_261i
Element source: ```<textarea id="sq_261i" class="spg-input spg-comment" rows="2" cols="50" placeholder="" aria-required="false" aria-labelledby="sq_261_ariaTitle" aria-describedby="sq_261_ariaDescription" aria-invalid="false" style="resize: none; height: 72px;"></textarea>```

![Image](https://github.com/user-attachments/assets/00546c05-7771-4f2d-ab77-f672661c1fc8)
